docs/data-sources/account_network_policies.md

```diff
@@ -2,6 +2,8 @@
 subcategory: "Settings"
 ---
 # databricks_account_network_policies Data Source
+Preview Stage: `GA`
```

Comment:
`Preview Stage` may not be the correct term - we use the term "platform release": https://docs.databricks.com/aws/en/release-notes/release-types#platform-releases... Maybe use "Release stage"?